rst_files/getting_started.rst

```diff
@@ -63,311 +52,52 @@ install the latest version `Anaconda <https://www.anaconda.com/download/#macos>`
-are writing `Markdown <http://daringfireball.net/projects/markdown/>`_, a mark-up language similar to (but simpler than) LaTeX
-
-(You can also use your mouse to select ``Markdown`` from the ``Code`` drop-down box just below the list of menu items)
-
-Now we ``Shift + Enter`` to produce this
-
-.. figure:: /_static/figures/nb7_julia.png
-   :scale: 100%
-
-
-Inserting unicode (e.g. Greek letters)
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-Julia supports the use of `unicode characters <https://docs.julialang.org/en/v1/manual/unicode-input/>`__
-such as ``α`` and ``β`` in your code
-
-Unicode characters can be typed quickly in Jupyter using the ``tab`` key
-
-Try creating a new code cell and typing ``\alpha``, then hitting the ``tab`` key on your keyboard
-
-Shell Commands
-^^^^^^^^^^^^^^^^
-
-You can execute shell commands (system commands) in Jupyter by prepending a semicolon
-
-For example, ``; ls`` will execute the UNIX style shell command ``ls``,
-which --- at least for UNIX style operating systems --- lists the
-contents of the current working directory
-
-These shell commands are handled by your default system shell and hence are platform specific
-
-
-Package Manager
-^^^^^^^^^^^^^^^^
-
-You can enter the package manager by prepending a ``]``
-
-For example, ``] st`` will give the status of installed packages in the current environment
-
-
-Sharing Notebooks
-------------------------
-
-Notebook files are just text files structured in `JSON <https://en.wikipedia.org/wiki/JSON>`_ and typically end with ``.ipynb``
-
-A notebook can easily be saved and shared between users --- you just need to
-pass around the ``ipynb`` file
-
-To open an existing ``ipynb`` file, import it from the dashboard (the first
-browser page that opens when you start Jupyter notebook) and run the cells or edit as discussed above
-
-The Jupyter organization has a site for sharing notebooks called `nbviewer <http://nbviewer.jupyter.org/>`_
-which provides a static HTML representations of notebooks
-
-.. Notebook can be downloaded as an ``ipynb`` file by clicking on the download icon at the top right of its page
-
-QuantEcon also hosts the `QuantEcon Notes <http://notes.quantecon.org/>`_ website, where you can upload and share your notebooks with other economists and the QuantEcon community
-
-.. _julia_repl:
-
-The REPL
-------------
-
-While we have not emphasized it, on any :ref:`JupyterHub <jl_jupyterhub>` or local Jupyter
-installation you will also have access to the Julia REPL
-
-This is a Julia specific terminal disconnected from the graphical interface of
-Jupyter, and becomes increasingly important as you learn Julia
-
-For example, the REPL is what we used in the beginning of this lecture to install
-``InstantiateFromURL.jl`` and ``IJulia``
+    ] add IJulia InstantiateFromURL
+    using InstantiateFromURL
+    activate_github("QuantEcon/QuantEconLecturePackages", tag = "v0.9.5", add_default_environment = true)
```

Comment:
I would kind of prefer users just installed IJulia and did the rest in the notebook or something. Or left the `InstantiateFromURL` stuff till later.

Reply:
This is intended to separate out the desktop installation from using the notebook stuff.  There is no jupyter used at all in this section (as it begins in the second lecture).

This all comes down to a decision of eitehr supporting juliabox/jupyterhub/etc. in the setup process or not.  If not, then the whole lectures would be reorganized.

Reply:
OK, we can do that. That would resolve the concern that @jlperla had, where we're putting important stuff about these commands in one specific section (the desktop install). 

So then we'll just leave all `activate_github` stuff until the next lecture (`julia_environments`)

Reply:
Not so fast.  The line of code is different for jupyterhub vs. desktop installations.  In fact, on juliabox or jupyterhub you wouldn't do this step (and not the `add_default_environment` for sure).

Do we ever have them boot up jupyter in this sub-lecture?

Reply:
I want to keep the jupyterhub vs. destkop things as segregated as possible.  Nobody using the desktop should have to worry about "cloud based options" and vice-versa.

Reply:
Also, I think it makes sense to put this here (at the end of `julia_environments`), because by this point they're all using Jupyter and can execute these steps in the Jupyter REPL.

Reply:
But, focusing entirely on the desktop installation, the question is whether we can do the activate_github in a notebook or not.  The reason to do it all in this section is so that all of the installation steps are all together, and afterwards they can start using everything.

@natashawatkins What I don't understand about your workflow is why it is easier on a desktop to copy/paste that text into a notebook vs. into a REPL?  Or maybe I am missing an installation step.  I don't envision people typing it.

Keep in mind here that they will not actually be using a notebook at this point because jupyter is not installed, so they are looking at this stuff on the web.

Reply:
> Also, I think it makes sense to put this here (at the end of julia_environments), because by this point they're all using Jupyter and can execute these steps in the Jupyter REPL.

Not unless we tell them to, and I am not sure why we should complicate things by having them start a notebook just to do that installation step?  But maybe I am missing something.

Reply:
@jlperla OK (sorry, for some reason your comments are loading asychronously). 

I think this approach (have the QuantEcon package stuff as a section, and have the local and cloud workflows be logically independent subsections) works?

Reply:
I was thinking that you don't really need these packages in this lecture to get started, but actually I guess it makes sense to install everything in this lecture.

Reply:
OK, caught up. I'll move this back to how it was. 

So, for clarity- 

1. In the `getting_started`, under the desktop installation, we have the:

```
using InstantiateFromURL 
activate_github(blahblah, add_default_environment = true) 
```

and explain it. 

2. In the same lecture, under the cloud stuff, we have the list of things to run through. That is: 

* Check that your setup doesn't come with a large list of packages. 
* If it doesn't, try adding `InstantiateFromURL`. If that doesn't work, try adding packages as you go. If that doesn't work, talk to a sysadmin. 
* If the `InstantiateFromURL` thing worked, then put the

```
using InstantiateFromURL 
activate_github("QuantEcon/QuantEconLecturePackages", tag = "v0.9.5")
```

and explain it a bit. 

3. Remove all mentions of the curated package setup from the second lecture, `julia_environments`.